configs/components/ruby-shadow.rb

```diff
@@ -15,7 +15,7 @@
     pkg.environment "RUBY", settings[:host_ruby]
     ruby = "#{settings[:host_ruby]} -r#{settings[:datadir]}/doc/rbconfig-#{settings[:ruby_version]}-orig.rb"
   else
-    ruby = File.join(settings[:bindir], 'ruby')
+    ruby = File.join(settings[:ruby_bindir], 'ruby')
```

Comment:
For most projects, `bindir` and `ruby_bindir` are the same directory, so it was not a problem to use `bindir` here, but for pe-bolt-server, these directories differ.